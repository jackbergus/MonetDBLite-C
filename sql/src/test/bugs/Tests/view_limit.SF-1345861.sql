create view foo as select * from ttables union all select * from ptables;
