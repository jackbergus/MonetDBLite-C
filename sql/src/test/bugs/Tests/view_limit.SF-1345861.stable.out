stdout of test 'view_limit.SF-1345861` in directory 'src/test/bugs` itself:


# 21:02:41 >  
# 21:02:41 >  Mtimeout -timeout 180 Mserver "--config=/ufs/niels/scratch/Linux-x86_64/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/ufs/niels/scratch/Linux-x86_64/lib/MonetDB:/ufs/niels/scratch/Linux-x86_64/lib/bin" --set "gdk_dbfarm=/ufs/niels/scratch/Linux-x86_64/var/MonetDB/dbfarm" --set "sql_logdir=/ufs/niels/scratch/Linux-x86_64/var/MonetDB/log" --set mapi_port=34977 --set sql_port=42163 --set xquery_port=58225 --set monet_prompt= --trace "--dbname=mTests_src_test_bugs" --dbinit="module(sql_server); sql_server_start();" ; echo ; echo Over..
# 21:02:41 >  

# Monet Database Server V4.9.3
# Copyright (c) 1993-2005, CWI. All rights reserved.
# Compiled for x86_64-redhat-linux-gnu/64bit with 64bit OIDs; dynamically linked.
# Visit http://monetdb.cwi.nl/ for further information.

printf("\nReady.\n");

Ready.
quit();

Over..

# 21:02:41 >  
# 21:02:41 >  Mtimeout -timeout 60 MapiClient -lsql -u monetdb -P monetdb --host=localhost --port=42163  < view_limit.SF-1345861.sql
# 21:02:41 >  

# 6 # querytype

# 21:02:41 >  
# 21:02:41 >  Done.
# 21:02:41 >  

